### PR TITLE
replacing multi-node with multi-role

### PR DESCRIPTION
**Pull request by vishalchangrani**
This PR updates the new content and replaces "multi-node" with "multi-role" as per this [discussion](https://flow-foundation.slack.com/archives/CCN7P6147/p1723073658025179)

### CODE REVIEW

File: docs/networks/network-architecture/index.md [hunk 1]
```diff
@@ -47,8 +53,6 @@ This division of labor between nodes occurs within the different validation stag
 In other words, every Flow node still participates in the validation of every transaction, but they do so only at one of the stages of validation.
 They can therefore specialize—and greatly increase the efficiency—for their particular stage of focus.
 
-![scenario_1](images/flow_node_types_1.gif)
-
 ### Flow node types and what they do
 
 |                                          |   Node type    | Responsibility                                                                                                                              | What do the nodes of this type do?                                                                                                                              |
```
Comment (AlexHentschel) [inline docs/networks/network-architecture/index.md:58]: Not sure ... wondering if we should use `node role` here too for consistency:

```suggestion
### Flow node roles and what they do

|                                          |   Node role    | Responsibility                                                                                                                              | What nodes with this role of do?                                                                                                                              |
```

**Review by AlexHentschel — APPROVED** [review]
thanks for the update. Very nice.

One optional thought  👉 [suggestion](https://github.com/onflow/docs/pull/844#discussion_r1714456274)

File: docs/networks/network-architecture/index.md (hunk 1 above)
Comment (vishalchangrani) [reply docs/networks/network-architecture/index.md:58]: I had set auto-merge on the PR and hence the changes went live...Will make another PR to address this comment.

Comment (AlexHentschel) [reply docs/networks/network-architecture/index.md:58]: sorry for the delayed commend. Thanks for the follow-up

Comment (vishalchangrani) [reply docs/networks/network-architecture/index.md:58]: https://github.com/onflow/docs/pull/850

**Pull request merged**